src/EA.Weee.Web/Areas/AatfReturn/Views/AatfTaskList/Index.cshtml

```diff
@@ -80,7 +80,15 @@
         <div class="govuk-grid-row govuk-table__cell weee-tonnage-top-aligned-display">
             <div class="govuk-grid-column-one-half">
                 <p>
-                    @Html.NavigationRouteLink("Sent to another AATF / ATF", @item.Aatf.Name + " sent to another AATF / ATF", AatfRedirect.AatfSelectedRoute, new { returnId = Model.ReturnId, aatfId = @item.Aatf.Id, action = "Index", controller = "SentOnCreateSite" })
+                    @if (Equals(item.WeeeSentOn.B2B, "-") && Equals(item.WeeeSentOn.B2C, "-"))
```

Comment:
So we can better test this, is it possible to add a property on the AatfsData item say bool RedirectSummary which set in the view model mapper, we can then create a unit test? Then use that bool value as the if statement.

Doing this in the View makes it harder to test.

Reply:
Done the refactoring and added a test.

Had to slightly change the test data to use the same Guid as an ID because things weren't matching up.